retroshare-gui/src/gui/MainWindow.cpp

```diff
@@ -1688,3 +1689,21 @@ void MainWindow::setCompactStatusMode(bool compact)
 	ratesstatus->setCompactMode(compact);
 	//opModeStatus: TODO Show only ???
 }
+
+// Called by other thread because Widgets must be created in the GUI thread.
+Gui_InputDialogReturn MainWindow::guiInputDialog(const QString& windowTitle, const QString& labelText, QLineEdit::EchoMode textEchoMode, bool modal)
+{
+
+	RsErr(this->thread(), " <-> ", qApp->thread());
```

Comment:
AFAIU you should use `RsDbg` or `RsInfo` here not `RsErr`

Reply:
Done